src/main/java/windycall/WindyCall.java

```diff
@@ -0,0 +1,76 @@
+package windycall;
+
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
+/**
+ * Serves as a trigger of the whole WindyCall chatBox application.
+ * Stores some important helper classes instance.
+ */
+public class WindyCall {
+
+      private List<Task> tasks;
+      private Storage storage;
+      private Ui ui;
```

Comment:
Having the Ui object be called ui and frequently calling both could be confusing. Might I suggest renaming the ui to uiObject? If you choose to do so, there is a feature in Intelij to automatically edit all instances of a phrase (shift+F6).

Reply:
Sure, it makes sense. Thanks for that.